Documentation/guides/messages/xa0111.md

```diff
@@ -17,14 +17,17 @@ aapt2 -version
 
 ## Solution
 
-Try re-installing the android build-tools.
+Check for a custom `<Aapt2ToolPath>` value in your _.csproj_ file and
```

Comment:
Another docs "convention" question (@brendanzagaeski?): should this use MSBuild property syntax `$(Aapt2ToolPath)` or XML syntax `<Aapt2ToolPath>`?

Reply:
The safest choice in terms of MSBuild property conventions would be a plain undecorated `Aapt2ToolPath`, but the way this sentence is phrased is a little special because it is talking about the XML content of the _.csproj_ file, so using the XML tag name instead of the MSBuild property name is not necessarily incorrect. One option could be to be explicit about the fact that it's an XML tag name:

```
Check for any `<Aapt2ToolPath>` elements in the _.csproj_ file and
remove them. Xamarin.Android redistributes its own recommended version
of AAPT2, so setting a custom value for the `Aapt2ToolPath` MSBuild
property is generally not recommended. Alternatively, try re-installing
the Xamarin workload in Visual Studio.
```

(This candidate wording also includes a little more detail for the second sentence about why it generally makes sense to remove this property.)

Example inspiration <https://docs.microsoft.com/dotnet/core/tools/csproj#sdk-attribute>:

> The root `<Project>` element of the _.csproj_ file has a new attribute called `Sdk`.